lib/dom_dsl_core.ml

```diff
@@ -32,7 +32,9 @@ module Common = struct
   end
 
   let fragment children = Core.Fragment.make ~children ()
+  let none = fragment []
   let string = Core.string
   let int = Core.int
   let float = Core.float
+  let maybe f = function Some value -> f value | None -> none
```

Comment:
I am not sure this helper is something that should be part of the DSL library scope 🤔 It seems a function that can be easily added to userland. Or maybe we can start a new library for this kind of helpers?

Reply:
It could, but then you have another module you'd have to open. It's also a bit inconsistent to provide this functionality for props, but not elements.

I don't feel very strongly about it, but I do think it'd be better to have this out of the box, assuming the design isn't too controversial and prone to change.